README.md

```diff
@@ -0,0 +1,24 @@
+# NoteApp 
+
+### Deployed Link
+https://noteapp-scrimba.netlify.app/
+
+
+Quick start:
+
+```
+$ yarn # npm install
+$ yarn build # npm run build
+````
```

Comment:
As you don't have any yarn.lock, which may allow installation with yarn package manager, it would be best to only write package installation instructions with npm, unless you also provide yarn.lock.

Reply:
done!